src/util/schema.py

```diff
@@ -278,6 +278,7 @@ class AmmKindEnum(str, Enum):
     CONSTANT_PRODUCT = "ConstantProduct"
     WEIGHTED_PRODUCT = "WeightedProduct"
     STABLE = "Stable"
+    CONCENTRATED = "Concentrated"
```

Comment:
Is it already parsed into something? If not, this should be added at some point, see #19

Reply:
Univ3 is not supported here, and I think it's a bit out of scope, at least for now.